live-streaming/live-stream-best-practices.md

```diff
@@ -38,49 +31,73 @@ After this, take network connection into consideration - if you or your users st
 
 Remember that both protocols have their pros and cons - with bad network conditions, viewers might encounter buffering with `RTMPS`, or they might encounter audio or visual glitches when using `SRT`.
 
-## DVR - reading history in live streams
+## Starting a live stream
 
-When a live stream is ongoing, viewers can replay earlier content with the DVR feature. The available buffer is 1 hour, and is only available in live streams that are actively broadcasting.
+- When a stream is starting it might take up to 30 seconds for the player to start delivering.
+- api.video recommends that you add an intro image or a simple countdown before starting the stream. This gives users time to connect before the stream starts, and makes it clear that the stream is about to begin.
+- To ensure that your live stream is running in optimal quality, check the [recommended settings](#recommended-setting-for-ingestion).
+- Make sure you verify that your connection is stable and the speed is adequate before starting the stream.
 
-When the stream ends, api.video stores the stream for another 4 to 5 minutes, subsequently ending the stream and disposing of the cached video.
+## Ending a live stream
 
-## Ending the live stream
+Make sure to use an outro like a static image or music at the end of the stream for about 30 seconds. This is a good way to indicate to the audience that the stream has ended. You can then request the API to complete the live stream, which stops the broadcast and the recording of the live stream as well. Read more about completing a live stream [here](/live-streaming/working-with-live-streams#complete-a-live-stream).
```

Comment:
Instead of `you can then request...` it would be better to say `you should then request...`

Reply:
Also, maybe use a new line regarding the sentence to complete the live to reduce the paragraph size.

Reply:
Fixed ✅